.gitattributes

```diff
@@ -1,6 +1,14 @@
-/tests export-ignore
-/spec export-ignore
-/.travis.yml export-ignore
-/.gitignore export-ignore
-/.gitattributes export-ignore
-/phpunit.xml.dist export-ignore
+/etc                        export-ignore
+/features                   export-ignore
+/spec                       export-ignore
+/tests                      export-ignore
+/.editorconfig              export-ignore
+/.gitattributes             export-ignore
+/.gitignore                 export-ignore
+/.scrutinizer.yml           export-ignore
+/.travis.yml                export-ignore
+/behat.yml.dist             export-ignore
+/easy-coding-standard.yaml  export-ignore
+/phpspec.yml.dist           export-ignore
+/phpstan.neon               export-ignore
+/README.md                  export-ignore
```

Comment:
I would keep the README :-)

Reply:
Why is that? :D

In my personal experience I have never, as in 0 times, read a readme in a vendor folder

Reply:
Haha, I do that :-)

Reply:
Crazy :D